docs/reference-guides/backup-restore-configuration/examples.md

```diff
@@ -244,6 +244,8 @@ spec:
 
 ## Example Credential Secret for Storing Backups in S3
 
+The `accessKey` and `secretKey` must be base64 encoded values. If you acquired your `accessKey` or `secretKey` through the Rancher UI, enter the value exactly as it appears in the UI. If you acquired either key through a commandline tool such as kubectl, you'll need to manually encode the key in base64.
```

Comment:
I don't fully agree with the concern brought up by #686 for this particular example since we're representing the secret using YAML here and the data field is base64 encoded per https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#secret-v1-core. Even if consider secret creation via "Edit as YAML" as using the UI, the input for `data` still needs to be base64 encoded. 

In any case, the current PR doesn't really address the concern brought up in #686 though. `The accessKey and secretKey must be base64 encoded values` is what #686 is suggesting isn't always true. The access key and secret key are referring to AWS keys, which you wouldn't obtain though the Rancher UI nor kubectl. I believe #686 is suggesting that we have something along the lines of:

If creating the secret using the UI (the form), enter the accessKey and secretKey as raw/plaintext.
If creating the secret using other methods, enter the accessKey and secretKey in their base64 encoded form.

But I think the simplest solution is to just remove the section from this page. The [backup-restore-configuration/backup-configuration](https://ranchermanager.docs.rancher.com/reference-guides/backup-restore-configuration/backup-configuration#example-credentialsecret) page already has this same sample YAML block, but with an explicit kubectl sample command on using it. We can amend that portion with a link to creating a secret though the UI as well. The section from this page is just a duplicate.

Reply:
Thanks for the thorough review. We should definitely reduce duplication across pages as that quickly adds up